rpcs3/Emu/RSX/VK/VulkanAPI.h

```diff
@@ -14,21 +14,6 @@
 #include "define_new_memleakdetect.h"
 #include "Utilities/types.h"
 
-// TODO: Remove when packages catch up, ubuntu is stuck at 1.1.73 (bionic) and 1.1.82 (cosmic)
-// Do we still use libvulkan-dev package on travis??????
-#if VK_HEADER_VERSION < 95
```

Comment:
Don't remove this, not all builds are done via travis/docker.

Reply:
The docker image `rpcs3/rpcs3-travis-trusty:1.1` uses `libvulkan1_1.1.106.0` / `libvulkan-dev_1.1.106.0`. Check this PR: https://github.com/hcorion/rpcs3-docker/pull/1

Reply:
Nvm, I get it. I will follow your advice

Reply:
Option 2: Remove this and add instructions in readme about adding the lunarg ppa for ubuntu users. Ubuntu LTS is still at 1.1.73.

Reply:
Option 1 done.